sklearn/feature_extraction/text.py

```diff
@@ -602,7 +602,8 @@ class HashingVectorizer(TransformerMixin, VectorizerMixin, BaseEstimator):
     ngram_range : tuple (min_n, max_n), default=(1, 1)
         The lower and upper boundary of the range of n-values for different
         n-grams to be extracted. All values of n such that min_n <= n <= max_n
-        will be used.
+        will be used. An ``ngram_range`` of (1, 2) means unigrams and bigrams,
```

Comment:
I think I would add double backticks around the tuple as well. And maybe add "for example" at the beginning, and also mention ``(1, 1)``?

Reply:
Thanks for your fast review! I incorporated your feedback and also found some minor issue at the default values in the ngram documentation.